src/vision/cmt_tracker/scripts/face_predictor.py

```diff
@@ -14,13 +14,14 @@
 from dynamic_reconfigure.server import Server
 
 from openface_wrapper import face_recognizer
-from image_scraper import image_scaper
+
 from pi_face_tracker.msg import FaceEvent
 class face_predictor:
     def __init__(self):
         rospy.init_node('face_recognizer', anonymous=True)
         self.image_scraper_enable = rospy.get_param('image_scraper')
         if self.image_scraper_enable != 0:
+            from image_scraper import image_scaper
```

Comment:
but this does not make sense -- you are merely moving the import to a different location; when `class face_predictor` gets instantiated, you'll just hit the very same code.  Why would you want to make the import conditional like this?